package.json

```diff
@@ -108,7 +108,7 @@
     "webpack": "1.13.1",
     "webpack-dev-middleware": "^1.6.1",
     "webpack-hot-middleware": "^2.12.0",
-    "workerjs": "^0.1.1",
+    "workerjs": "github:jlongster/workerjs",
```

Comment:
👍  we should probably file a pr even though it's stale. atleast others will see that we've taken a stab at it